Source/Fuse.Controls.CameraView/Android/CameraImpl.java

```diff
@@ -8,21 +8,27 @@
 import android.hardware.Camera.Size;
 import android.view.Surface;
 import java.util.List;
+import java.util.ArrayList;
 import android.graphics.SurfaceTexture;
 import android.graphics.Matrix;
+import android.graphics.Rect;
 import android.media.MediaRecorder;
 import android.view.OrientationEventListener;
+import android.util.Log;
+import android.view.Display;
 
 public class CameraImpl extends TextureView implements TextureView.SurfaceTextureListener {
 
     final Camera _camera;
-    final boolean _autoFocus;
+    boolean _autoFocus;
     final int _maxWidth;
     final int _maxHeight;
     final int _cameraId;
     final OrientationEventListener _orientationListener;
 
     int _cameraRotation = 0;
+    int previewWidth;
+    int previewHeight;
```

Comment:
Private fields start with an underscore, according to the coding standard...

Reply:
Oh hey, I just realised, that those are not my variables!

Reply:
...I shall change them for you while I'm at it ;)